### PR TITLE
docs: Updated Installation Guide

### PR DESCRIPTION
**Pull request by ehino**
Signed-off-by: Ehigiator Akhile <akhileehis@gmail.com>

### Related Issue

Solving Issue: #xxx

### PR Type

- [ ] Fix typos or format(punctuation, space, indentation, code block, etc.)
- [ ] Update inappropriate or misunderstanding content
- [x] Add missing content or new documents
- [ ] Delete useless content or outdated documents
- [ ] Localize or translate documents
- [ ] Deal with GitHub action files and scripts
- [ ] Deal with chores

### CODE REVIEW

File: content/getting-started/installation.mdx [hunk 1]
```diff
@@ -39,12 +39,34 @@ kubectl apply -f https://download.linkall.com/all-in-one/v0.5.3.cn.yml
...
+```
+
+Now, you can easily see the pods in the namespace
 
 ```shell
-kubectl get pods -n vanus
+kubectl get pods
 ```
 
+
```
Comment (JieDing) [inline content/getting-started/installation.mdx:69]: If you don't want to change the default namespace, you can use `-n vanus` to declare which namespace you are trying to access.

```shell
kubectl get pods -n vanus
```

Comment (JieDing) [reply content/getting-started/installation.mdx:69]: @ehino See comment

Comment (ehino) [reply content/getting-started/installation.mdx:69]: Sure sure
Thanks!

**Review by JieDing — APPROVED** [review]
LGTM

**Pull request merged**